web/src/ui_init.js

```diff
@@ -220,6 +220,32 @@ function initialize_right_sidebar() {
             }
         }
     });
+
+    $("#narrow-users-presence-container").on("click", ".buddy-list-subsection-header", () => {
+        $("#narrow_user_presences").toggleClass("no-display");
+        const is_collapsed = $("#narrow_user_presences").hasClass("no-display");
```

Comment:
`no-display` feels like a little bit non-semantic as a class name; do we want `collapsed` or `presences-closed` or something along those lines instead?